recipes/make/all/test_package/conanfile.py

```diff
@@ -0,0 +1,11 @@
+from conans import ConanFile, AutoToolsBuildEnvironment, tools
+import os
+
+
+class TestPackageConan(ConanFile):
+    settings = "os", "compiler", "build_type", "arch"
+
+    def test(self):
+        with tools.chdir(self.source_folder):
+            env_build = AutoToolsBuildEnvironment(self)
```

Comment:
Could you test here CONAN_MAKE_PROGRAM value?

Reply:
Maybe also a `which make` and check that the path is inside a conan cache? not sure that would feasible

Reply:
maybe it's better to use [tools.remove_from_path](https://docs.conan.io/en/latest/reference/tools.html#tools-remove-from-path) then?